galleries/examples/event_handling/figure_axes_enter_leave.py

```diff
@@ -42,7 +42,7 @@ def on_leave_figure(event):
 
 
 fig, axs = plt.subplots(2, 1)
-fig.suptitle('mouse hover over figure or axes to trigger events')
+fig.suptitle('mouse hover over figure or Axes to trigger events')
```

Comment:
I know the focus was on `axes` vs `Axes`, but feels incongruent to leave `figure` uncapitalized when `Axes` is capitalized

This was the first place I noticed it (because this was the first line in the diff as rendered on GitHub), but that thought did come up in several other files, notably `legend.py` and `patches.py`, and several other places where figures are referred to in the same manner as `Axes`, but the capitalization is different.


Also this is in galleries, fyi.

Reply:
Moved to #27104.

Reply:
Lower case figure: See https://github.com/matplotlib/matplotlib/pull/18707#discussion_r504284535 and the response https://github.com/matplotlib/matplotlib/pull/18707#discussion_r504403247

Reply:
The response is that `Figure` _may be_ `figure` without ambiguity, not that it _must be_ `figure`.